### PR TITLE
no really who is da king

### PR DESCRIPTION
**Pull request by Bawhoppen**
:cl: 
add: Added crowns. Make them from gold. Cap gets a fancy one.
/:cl:

ss 2016-07-27 at 11 42 11

Crowns get some armor but are overall worse than sec gear.
~~Swat gear now has better armor because they weren't any better than standard sec gear after steelpoint's buff. 
Fixed the vest being unequal armor from the helmet (I know it was because of the sec jumpsuit but, the jumpsuit armor is a bonus, not supposed to be factored into the vest's armor stats)~~

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -33,7 +33,7 @@
 	icon_state = "hos"
 	item_state = "greatcoat"
 	body_parts_covered = CHEST|GROIN|ARMS|LEGS
-	armor = list(melee = 30, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0)
+	armor = list(melee = 40, bullet = 30, laser = 30, energy = 10, bomb = 25, bio = 0, rad = 0)
```
Comment (GunHog) [inline code/modules/clothing/suits/armor.dm]: Security armor changes need to be their own PR, not bundled with a new item only loosely related to Sec.

Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: okay then this is a PR for sec armor changes and the crown is an additional thing easy

Comment (KorPhaeron) [inline code/modules/clothing/suits/armor.dm]: A T O M I Z E

Comment (Bawhoppen) [inline code/modules/clothing/suits/armor.dm]: its so minor

**Comment by Bawhoppen** [comment]
ITS SO MINOR THOUGH

**Comment by Bawhoppen** [comment]
there fixed

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 2]
```diff
@@ -12,6 +12,7 @@
 		new /obj/item/weapon/storage/backpack/satchel/cap(src)
 	new /obj/item/clothing/suit/cloak/cap(src)
 	new /obj/item/weapon/storage/backpack/dufflebag/captain(src)
+	new /obj/item/clothing/head/crown/fancy(src)
```
Comment (KorPhaeron) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: my immersions

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 3]
```diff
@@ -181,6 +181,7 @@ var/global/list/datum/stack_recipe/gold_recipes = list ( \
 	new/datum/stack_recipe("HoP Statue", /obj/structure/statue/gold/hop, 5, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("CE Statue", /obj/structure/statue/gold/ce, 5, one_per_turf = 1, on_floor = 1), \
 	new/datum/stack_recipe("RD Statue", /obj/structure/statue/gold/rd, 5, one_per_turf = 1, on_floor = 1), \
+	new/datum/stack_recipe("Simple Crown", /obj/item/clothing/head/crown, 5), \
```
Comment (KorPhaeron) [inline code/game/objects/items/stacks/sheets/mineral.dm]: It's' funny that you can build five guns for the cost of one crown (like 20 guns if you upgrade the RD machines), but that's more RD's costs being a joke than your fault.

**Comment by Bawhoppen** [comment]
fine atomized fucking cuckmongers

File: code/game/objects/items/stacks/sheets/mineral.dm (hunk 3 above)
Comment (Bawhoppen) [inline code/game/objects/items/stacks/sheets/mineral.dm]: to be fair guns theres other materials in one gun

**Comment by PKPenguin321** [comment]
good shit baw. this game needs more dope bling :+1:

**Comment by Bawhoppen** [comment]
oh sprites by WJ btw

**Comment by KorPhaeron** [comment]
>captain now starts with a sword and crown to lead the crew in fights against cults, devils, and dragons

coreoverload was right

**Comment by Alek2ander** [comment]
:crown: LONG MAY HE REIGN :crown:

**Comment by Core0verload** [comment]
:+1: to making crowns out of gold
:-1: to roundstart crowns in lockers

**Comment by GunHog** [comment]
Who cares if the Captain wants to play Pretend King with this saber and crown? He will just get himself slipped by the clown, then murdered and looted by Spear-Assistants.

**Comment by Niknakflak** [comment]
onmob sprites for the crowns aren't even that good

**Comment by Alek2ander** [comment]
metalthrone

**Comment by Bawhoppen** [comment]
why is there still the DNM tag

File: code/modules/clothing/head/misc.dm [hunk 4]
```diff
@@ -259,4 +259,15 @@
...
 	icon_state = "papersack_smile"
-	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
\ No newline at end of file
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+
+/obj/item/clothing/head/crown
+	name = "crown"
+	desc = "A crown fit for a king, a petty king maybe."
+	icon_state = "crown"
+	armor = list(melee = 50, bullet = 0, laser = 0,energy = 20, bomb = 0, bio = 0, rad = 0) //for a warrior king //energy cause gold is reflective //worse than sec helm overall
```
Comment (optimumtact) [inline code/modules/clothing/head/misc.dm]: 50 melee resist on a stupid item

This is a new low.

Remove this please, because it makes it easier for steelpoint to justify his security armour buffs.

Comment (Bawhoppen) [inline code/modules/clothing/head/misc.dm]: I wanted it to have slightly better melee than sechelms, but not have any of the other benefits of sechelms so it would be overall worse

Comment (optimumtact) [inline code/modules/clothing/head/misc.dm]: I don't care what you want

Comment (Bawhoppen) [inline code/modules/clothing/head/misc.dm]: okay don't then

Comment (Bawhoppen) [inline code/modules/clothing/head/misc.dm]: okay listen oranges, imagine a raid group of people all wearing badass crowns rushing into security fighting people with spears

Comment (optimumtact) [inline code/modules/clothing/head/misc.dm]: That can already happen, what concerns me is that lots of little items like this creep into the game, everyone sees that so much stuff has near the same armour counts as actual armour, so some bright chap gets it in their head to come along and double all the sec armour counts to make them stand out.

then the cycle continues in the eternal buff and any attempt we make to nerf it just leads to a big slap fight and please I dont' want to go through that again just make it stop make the pain stop I can'ts leep I'm getting ulcers please it's easier to buff things than it is to nerf them always start out by making the item weak and then buffing it later based on feedback.

Comment (Bawhoppen) [inline code/modules/clothing/head/misc.dm]: In this case I think you're just paranoid

Comment (Niknakflak) [inline code/modules/clothing/head/misc.dm]: baw btfo

Comment (PKPenguin321) [inline code/modules/clothing/head/misc.dm]: The phenomenon oranges described is often called "powercreep" and yes it does happen

Comment (Bawhoppen) [inline code/modules/clothing/head/misc.dm]: powercreep does exist, but what oranges described for this specific scenario really won't be a problem

Comment (optimumtact) [inline code/modules/clothing/head/misc.dm]: Just remove the armour so I can merge this already.

**Comment by jughu** [comment]
Only you add a king cloak sometime later

**Comment by Bawhoppen** [comment]
captain's cloak is basically kingcloak

**Comment by jughu** [comment]
Make it fancy like usual kings cloak with fur n stuff

**Comment by optimumtact** [comment]
This is do not merge until you remove the armour, stop abusing the maintainers good nature.

**Comment by Bawhoppen** [comment]
fuck off oranges thats not why the DNM tag was added

**Comment by KorPhaeron** [comment]
listen to oranges

**Comment by Bawhoppen** [comment]
oranges literally does not have an argument for this that makes sense from any angle

**Comment by optimumtact** [comment]
your fluff item has more melee head protection that a hardhat, 3/4 of the melee protection of a security helmet, more than a roman helmet, more than the detectives hat, more than the hops cap, more than a welding mask.

It is absolute rediculous for an item that is for playing dressup has that much melee protection.

Stop trying to pretend that this is some kind of vendetta.

**Comment by Bawhoppen** [comment]
I explained all my reasoning in the coderbus, I even offered to lower it to next to nothing yet you don't seem to care, it isn't about the item itself, it is because I made the PR

**Comment by optimumtact** [comment]
what about 15 then same as the hardhat?

**Comment by Bawhoppen** [comment]
i offered lower than that but ok sounds good2me

**Comment by Bawhoppen** [comment]
me and my best bud oranges :smile_cat:
